<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="styles.css">
    <title>Exception Handling in JS</title> 
</head>

<body>
    <nav>
        <h2>Java Script | Exception Handling</h2>
    </nav>
    <script>
      //addNum(10,20);
      /*
      let n = 12;
      console.log(n.toUpperCase())
      */
      /*
      let n = 12.34252525;
      console.log(n.toPrecision(200))      
      console.log(`--- DONE ---`)
      */
    </script>
    <script>
      //let addNum = ()=>console.log('Inside addNum')
      try{
        addNum();
      }
      catch(ex){
        console.error(`Exception Occured!!`)
        console.error(`Error Type: ${ex.name} | Message: ${ex.message}`)
      }
      finally{
        console.log(`Application Designed & Developed by `);
        console.log(`team @ wipro`)
      }
      console.log(`--- DONE ---`)
    </script>
</body>
</html>